packages/payment-processor/README.md

```diff
@@ -6,6 +6,20 @@ It contains client-side payment methods for:
 - ERC20 proxy contract
 - ETH input data
 
+### Test
+
```

Comment:
Maybe a quick line to explain why we do it could be nice. Like:
```suggestion
To run the payment-processor tests we need a local running ganache with all our smart contracts deployed. You can open two terminals and do:
```